Makefile

```diff
@@ -5,7 +5,7 @@ SHELL := /bin/bash
 NO_CACHE = $(shell if [ $${NO_CACHE:-false} != false ]; then echo --no-cache; fi)
 SUDO := $(shell if docker info 2>&1 | grep "permission denied" >/dev/null; then echo "sudo -E"; fi)
 DOCKER := $(SUDO) docker
-DOCKER_IMAGE := binaris
+DOCKER_IMAGE := binaris/binaris
```

Comment:
starting to think it's time to rename this repo to `cli`. `binaris/binaris` is just too weird.

Reply:
Agreed. https://github.com/binaris/binaris/issues/271